app/controllers/avo/base_controller.rb

```diff
@@ -139,7 +135,6 @@ def create
           related_record = related_resource.find_record params[:via_record_id], params: params
 
           @record.send("#{@reflection.foreign_key}=", related_record.id)
```

Comment:
Hmmm. Wouldn't this break if this is a new record and it's not persisted to the DB?

```ruby
user = User.new
user.post = Post.new # wouldn't this break if there were a FK constraint or validation rule?
user.save
```

Reply:
Actually this is being done:

```ruby
user = User.new
user.post_id = 4 # random id
user.save
```

`related_record` is already persistent

Reply:
Got it! Then it's good to merge!